### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by AbhiGattineni**
Created the signup page.

### CODE REVIEW

File: js/phoenix/src/pages/signup.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { SignupForm } from "../containers";
+import { BareIcon } from "../../src/components"
```
Comment (RushilBasappa) [inline js/phoenix/src/pages/signup.jsx:2]: @AbhiGattineni replace with "../components"

File: js/phoenix/src/pages/signup.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { BsLinkedin, BsGoogle, BsGithub } from "react-icons/bs";
+
+const Signup = () => {
+    return (
+        <div className="flex flex-row">
+            <div className="basis-1 md:basis-2/3  flex flex-col justify-center items-center">
+                <h1 className="font-bold text-3xl">Add Details to Signup</h1>
+                <SignupForm />
+                <div className="flex items-center">
+                    <div className="border-t-2 border-gray-400  w-40"></div>
```
Comment (RushilBasappa) [inline js/phoenix/src/pages/signup.jsx]: @AbhiGattineni separate this into a component

File: js/phoenix/src/pages/signup.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                    <div className="m-4">OR</div>
+                    <div className="border-t-2 border-gray-400  w-40"></div>
+                </div>
+                <div className="font-bold text-xs">Signup with</div>
+                <div className="flex flex-row">
+                    <BareIcon IconComponent={<BsGoogle />} classes={["text-red-500"]} />
```
Comment (RushilBasappa) [inline js/phoenix/src/pages/signup.jsx:15]: @AbhiGattineni remove this color red

**Review by RushilBasappa — CHANGES_REQUESTED** [review]

File: js/phoenix/src/pages/signup.jsx (hunk 1 above)
Comment (AbhiGattineni) [reply js/phoenix/src/pages/signup.jsx:2]: replaced with components.

File: js/phoenix/src/pages/signup.jsx (hunk 2 above)
Comment (AbhiGattineni) [reply js/phoenix/src/pages/signup.jsx]: Separated and updated in the signup, and login pages.

File: js/phoenix/src/pages/signup.jsx (hunk 3 above)
Comment (AbhiGattineni) [reply js/phoenix/src/pages/signup.jsx:15]: removed red color

File: js/phoenix/src/pages/login.jsx [hunk 4]
```diff
@@ -1,23 +1,21 @@
...
       <div className="md:w-2/3 flex flex-col justify-center items-center">
         <h1 className="font-bold text-3xl">Login to Your Account</h1>
-        <LoginForm/>
-        <div className="flex items-center">
-          <div className="border-t-2 border-gray-400  w-40"></div>
-          <div className="m-4">OR</div>
-          <div className="border-t-2 border-gray-400  w-40"></div>
-        </div>
+        <LoginForm />
+        <Separater />
```
Comment (RushilBasappa) [inline js/phoenix/src/pages/login.jsx:13]: @AbhiGattineni Misspell *Separator

**Review by StephenJakku — APPROVED** [review]

**Review by RushilBasappa — APPROVED** [review]

**Pull request merged**